Bundle/CoreBundle/Resources/translations/victoire.en.xliff

```diff
@@ -224,7 +224,7 @@
       </trans-unit>
       <trans-unit id="58893b995da32" resname="victoire.widget.modal.title">
         <source>victoire.widget.modal.title</source>
-        <target>Widget %id% (%type%)</target>
+        <target>Widget %id%(%type%)</target>
```

Comment:
Widget modal heading doesn't have a widget id when you create a new Widget, so we don't want any '#' :
New widget : Widget (Plain Text)
Edit widget : Widget #1 (Plain Text)